app/3.0/docs/devguide/data-binding.md

```diff
@@ -460,36 +490,30 @@ backslash (`\`).
 
 Example:
 
-```html
-<dom-module id="x-custom">
-  <template>
-    <span>{{translate('Hello\, nice to meet you', first, last)}}</span>
-  </template>
-</dom-module>
+```js
+static get template() {
+  return html`
+    <span>{{translate('Hello\, nice to meet you', given, family)}}</span>
+  `;
+}
 ```
 
 Finally, if a computed binding has no dependent properties, it is only evaluated once:
 
-```
-<dom-module id="x-custom">
-  <template>
-    <span>{{doThisOnce()}}</span>
-  </template>
-
-  <script>
-    class XCustom extends Polymer.Element {
-
-      static get is() {return 'x-custom'}
-
-      doThisOnce: function() {
-        return Math.random();
-      }
-
-    }
+[See it in Plunker](https://plnkr.co/edit/iWFpsh?p=preview)
```

Comment:
Plunker here still has the name being set in index.html, which doesn't cause any problems, but is kind of confusing since this element doesn't do anything with the name.